<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrincipalWindow</class>
 <widget class="QMainWindow" name="PrincipalWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="mostrar_foto">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>531</width>
      <height>331</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_buscar">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>50</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Buscar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="agregar_puesto">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>530</y>
      <width>191</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar nuevo puesto de comida</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="buscador">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>461</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="mostrar_correo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>661</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 16pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Bienvenido: </string>
    </property>
   </widget>
   <widget class="QLabel" name="mostrar_resultado">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>390</y>
      <width>531</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_siguiente">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>230</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Siguiente</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_cerrar">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Cerrar sesi√≥n</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lista_comentarios">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>721</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_anterior">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Anterior</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_perfil_puesto">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>530</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Perfil del puesto</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
